lang/tests/src/test/scala/com/wavesplatform/lang/evaluator/BytesDropTakeTest.scala

```diff
@@ -0,0 +1,76 @@
+package com.wavesplatform.lang.evaluator
+
+import com.wavesplatform.common.state.ByteStr
+import com.wavesplatform.lang.directives.values.{V1, V5, V6}
+import com.wavesplatform.lang.v1.compiler.Terms.CONST_BYTESTR
+import com.wavesplatform.test._
+
+class BytesDropTakeTest extends EvaluatorSpec {
+  private val min   = Long.MinValue
+  private val max   = Long.MaxValue
+  private val limit = 165947
+
+  private val b = s"base58'${ByteStr.fromBytes(1, 2, 3, 4, 5)}'"
+
+  private def bytes(bytes: Byte*) = CONST_BYTESTR(ByteStr.fromBytes(bytes: _*))
+
+  property("take") {
+    eval(s""" take($b, 0) """) shouldBe bytes()
+    eval(s""" take($b, 2) """) shouldBe bytes(1, 2)
+    eval(s""" take($b, 100) """) shouldBe bytes(1, 2, 3, 4, 5)
```

Comment:
Use `$limit` instead of `100` to test that the limit itself is included

Reply:
Done